internal/home/control.go

```diff
@@ -346,6 +346,12 @@ func handleHTTPSRedirect(w http.ResponseWriter, r *http.Request) (ok bool) {
 		Scheme: aghhttp.SchemeHTTP,
 		Host:   r.Host,
 	}
+
+	if config.DNS.ServeHTTP3 {
+		altSvc := fmt.Sprintf(`h3=":%[1]v"; ma=2592000,h3-29=":%[1]v"; ma=2592000,h3-Q050=":%[1]v";ma=2592000,h3-Q046=":%[1]v"; ma=2592000,h3-Q043=":%[1]v"; ma=2592000,quic=":%[1]v"; ma=2592000; v="46,43"`, config.TLS.PortHTTPS)
```

Comment:
1.  This string is against our [line length guidelines](https://github.com/AdguardTeam/CodeGuidelines/blob/master/Go/Go.md#li-6e457108).

2.  Do we really need to enumerate all these drafts?  Wouldn't just `h3` suffice?

3.  I'd prefer `ma` to be set by default to a lower value, for example 24 h.  A TODO about considering making it configurable would be nice as well.

4.  Shouldn't this header be set on every response?  Because the way this is written, it is not set when redirecting to the HTTPS version.